examples/pcc-transport-chatbot/client-react/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "my-nextjs-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@pipecat-ai/client-js": "^0.3.5",
+    "@pipecat-ai/client-react": "^0.3.5",
+    "@pipecat-ai/daily-transport": "^0.3.7",
```

Comment:
We should probably have something similar for the client as well, a wrapper for all the available transports, like `pcc-transport`.